tests/integration-tests/tests/common/utils.py

```diff
@@ -90,11 +90,56 @@ def retrieve_latest_ami(region, os, ami_type="official", architecture="x86_64"):
         sys.exit(1)
 
 
+def retrieve_pcluster_ami_without_standard_naming(region, os, version="2.8.1", architecture="x86_64"):
```

Comment:
This is very specific to `test_create_wrong_pcluster_version`, should we move this function into test create file rather than keeping in common util?